pkg/provisioner/k8sprovisioner.go

```diff
@@ -120,6 +122,12 @@ func (p k8sProvisioner) newWorkerPod(args WorkerArgs) v1.Pod {
 		MountPath: "/root/.ssh",
 	})
 
+	gitVolumeMounts = append(gitVolumeMounts, v1.VolumeMount{
+		Name:      certVolumeName,
+		ReadOnly:  true,
+		MountPath: certVolumeMountPath,
+	})
```

Comment:
Need to mount the certs inside the docker step pod in k8spodtemplating.go instead of on the worker pod here in k8sprovisioner.go